coc/wars.py

```diff
@@ -179,11 +179,12 @@ def type(self):
         """
         if not self.start_time:
             return None
+        prep_list = [900, 1800, 3600, 7200, 14400, 21600, 28800, 43200, 57600, 72000, 86400]
```

Comment:
This list may appear a bit random on first glance- it might be a bit clearer if it were either something like 
```py
friendly_prep_times = [60*60*0.25, 60*60*0.5, 60*60, 60*60*2 ...]
``` 
and/or including a comment describing why, possibly in the docstring too